### PR TITLE
Create 2024-03-26.md

### PR DESCRIPTION
**Pull request by robcraig-LF**
Adds meeting minutes and agenda from SPDX Tech Team Meeting 2024-03-26

### CODE REVIEW

File: tech/2024-03-26.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+   - Now on OMG web site as request for comment - please add to SPDX project issues/trackers.   Please fix in linux foundation processes.   As long as nothing that can't be addressed in finalization task force, we'll be good.
+   - Next member meeting in June, and make sure are addressable.   RFC version, and can be addressed.   Bring together,  and an actual final spec come out.    Know revs since RC2 version for OMG. 
+   - Need to produce the format as ISO standard - from web version as well. 
+   - Jeff asks for specifications for exactly what need,  Bob has LATEX based markdown, and let them working about formating.  Bob to follow up on this.   Couple sections missing;   Bob to send to Jeff things to fold back in.   Jory says that https://www.metanorma.org/author/iso/.  Architecture board is trying to be accomodating - ie. using TRTL rather than MOF.
+   - Kate want to see if we can say 3.0 is out at OSS NA.   Bob thinks that that should be possible.   RDF & schemas should be locked thought.
```
Comment (goneall) [inline tech/2024-03-26.md:48]:
```suggestion
   - Kate want to see if we can say 3.0 is out at OSS NA.   Bob thinks that that should be possible.   RDF & schemas should be locked though.    
```

**Review by goneall — COMMENTED** [review]
One typo fix -

@kestewart - please review as I was not in the meeting myself

**Comment by kestewart** [comment]
Thanks Gary.  Change applied.

**Review by kestewart — APPROVED** [review]
LGTM.

**Pull request merged**